### PR TITLE
move three .svg to refer them easily

### PR DESCRIPTION
**Pull request by StevenJokes**
*Description of changes:*

By submitting this pull request, I confirm that you can use, modify,
copy, and redistribute this contribution, under the terms of your
choice.

### CODE REVIEW

**Comment by mli** [comment]
Job d2l-en/PR-1171/1 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1171/

**Comment by mli** [comment]
Is there any particular reason to move the SVGs? It may cause issues for the figure positions in the PDF

**Comment by StevenJokes** [comment]
We can refer them more easily while we reading. When we read the the old version, I can't find required pictures in one screen.
image
image
image

**Comment by mli** [comment]
@astonzhang  please check if it affects the PDF version

File: chapter_computational-performance/multiple-gpus.md [hunk 1]
```diff
@@ -27,23 +28,23 @@ Let us start with a simple computer vision problem and a slightly archaic networ
...
-
 By and large, data parallelism is the most convenient way to proceed, provided that we have access to GPUs with sufficiently large memory. See also :cite:` Li.Andersen.Park.ea.2014` for a detailed description of partitioning for distributed training. GPU memory used to be a problem in the early days of deep learning. By now this issue has been resolved for all but the most unusual cases. We focus on data parallelism in what follows.
 
 ## Data Parallelism
 
 Assume that there are $k$ GPUs on a machine. Given the model to be trained, each GPU will maintain a complete set of model parameters independently. Training proceeds as follows (see :numref:`fig_data_parallel` for details on data parallel training on two GPUs):
 
+![Calculation of minibatch stochastic gradient using data parallelism and two GPUs. ](../img/data-parallel.svg)
+:label:`fig_data_parallel`
+
```
Comment (astonzhang) [inline chapter_computational-performance/multiple-gpus.md:39]: We should avoid inserting a figure in the middle of a paragraph. Can you move this figure just above the subtitle "Data Parallelism"?

**Pull request merged**